recipes/llm-voice-assistant/python/main.py

```diff
@@ -59,72 +61,72 @@ def tps(self) -> float:
 def orca_worker(access_key: str, connection, warmup_sec: float, stream_frame_sec: int = 0.03) -> None:
     # noinspection PyUnresolvedReferences
     import numpy as np
-    from sounddevice import OutputStream
```

Comment:
Can we import it in the global scope? For `sounddevice` I couldn't because otherwise the device was owned by the wrong process.